### ISSUE TITLE
Compile error for `--bin=stress`

### ISSUE THREAD

**Issue by jeehoonkang** [spacejam/rsdb#113]
# meta

```
$ rustc --version
rustc 1.22.0-nightly (f861b6ee4 2017-09-01)
```

# error message

```
$ cargo build --bin=stress --features=stress,zstd; hack/flamerun.sh ./target/debug/stress --duration 5
   Compiling rsdb v0.12.0 (file:///home/jeehoonkang/Works/rsdb)
error[E0599]: no method named `__delete_all_files` found for type `std::sync::Arc<rsdb::Tree>` in the current scope
   --> src/bin/stress.rs:140:10
    |
140 |     tree.__delete_all_files();
    |          ^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

error: Could not compile `rsdb`.

To learn more, run the command again with --verbose.
+ perf record -F 99 -g ./target/debug/stress --duration 5
Workload failed: No such file or directory
```

**Comment by spacejam** [comment]
Whoops, forgot to clean that up! Thanks!

**Comment by spacejam** [comment]
This should be fixed now on `0.12.1`!
